### PR TITLE
feat(ui): add an icon chatbot

### PR DESCRIPTION
**Pull request by JulienMassonnat**
I added a small rounded icon on the right bottom corner to link to our OKP4GPT. 
I've also added some smooth animations. Below you can find the icon in a screencapture and a short video with the animations.

image
https://github.com/okp4/okp4-web/assets/63841183/8e6ac26a-d1a3-4845-b950-fd34e98263bb

### CODE REVIEW

**Review by MdechampG — APPROVED** [review]
Great ! Just a little remark : it's a bit strange that it continuously blurring and unblurring when the mouse is on the logo. Maybe it's possible not to redo indefinitely the animation ?

**Comment by JulienMassonnat** [comment]
Is it better ? 

https://github.com/okp4/okp4-web/assets/63841183/5184c7a5-9893-40d1-ba4e-7b8ec87eee4a

**Comment by MdechampG** [comment]
> Is it better ?
> 
>  2023-12-18.14-35-36.mp4

Yes it's better thank you !

**Comment by JulienMassonnat** [comment]
I added a new commit to modify with Marine's comment

File: src/assets/styles/transversals/_footer.scss [hunk 1]
```diff
@@ -51,6 +51,93 @@
...
+  right: 30px;
+}
+
+.chatBotImage {
+  cursor: pointer;
+  transition: all 0.5s ease;
+  border: solid 5px white;
+  opacity: 0.5;
+  border-radius: 100px;
+  position: relative; /* Ajout pour le positionnement du texte */
```
Comment (ccamel) [inline src/assets/styles/transversals/_footer.scss:106]: Remove all the comments (unecessary).

File: src/assets/styles/transversals/_footer.scss [hunk 2]
```diff
@@ -51,6 +51,114 @@
...
+  opacity: 0;
+  visibility: hidden;
+  opacity: 0;
+  position: absolute;
+  bottom: 100%; /* Départ de l'animation depuis le bas de l'image */
+  left: 50%;
+  transform: translateX(-80%);
+  transition: visibility 0s, opacity 0.5s linear;
+  animation:blurEffect 2s ease-in-out;
+}
```
Comment (ccamel) [inline src/assets/styles/transversals/_footer.scss:149]: Remove duplicated properties.

```suggestion
.chatBotText {
  font-family: 'Norse';
  font-size: 25px;
  width: 120px;
  color: white;
  text-align: center;
  border-radius: 5px;
  padding: 5px 0;
  z-index: 99;
  margin-left: -60px;
  display: inline-block;
  visibility: hidden;
  opacity: 0;
  position: absolute;
  bottom: 100%;
  left: 50%;
  transform: translateX(-80%);
  transition: visibility 0s, opacity 0.5s linear;
}
```

**Review by ccamel — CHANGES_REQUESTED** [review]
Some minor remarks.

File: src/assets/styles/transversals/_footer.scss (hunk 1 above)
Comment (JulienMassonnat) [reply src/assets/styles/transversals/_footer.scss:106]: Ok thanks

File: src/assets/styles/transversals/_footer.scss (hunk 2 above)
Comment (JulienMassonnat) [reply src/assets/styles/transversals/_footer.scss:149]: Ok thanks ;)

File: src/components/Footer.js [hunk 3]
```diff
@@ -70,6 +71,14 @@ const Footer = ({
           <span>Back to top</span>
         </div>
       </div>
+      <div
+        className="chatBotOracle"
+        role="button"
+        tabIndex={0}
+        onKeyDown={handleKeyDown}
```
Comment (ccamel) [inline src/components/Footer.js:78]: You should remove this line.

**Comment by JulienMassonnat** [comment]
Thanks @ccamel I changed the behavior.
On smartphone it's displayed only inside the burger menu
image
On desktop it's an icon chatbot as seen previously

**Review by ccamel — APPROVED** [review]
Awesome! :+1:

**Pull request merged**